values.yaml

```diff
@@ -447,6 +453,40 @@ clair:
   ## Additional deployment annotations
   podAnnotations: {}
 
+trivy:
+  # enable is the flag to enable Trivy scanner
+  enabled: true
```

Comment:
Both trivy and clair are enabled by default, is this by design?

Reply:
I think it should be Trivy and Clair can be both deployed as default scanners. Under this situation, Trivy is the default, Clair is not.

For enablement, I think Trivy should be enabled as 1st choice and Clair as a second choice which needs users to explicitly specify `enabled:true`.

Reply:
Both trivy and clair are enabled, trivy as default.